liquid_node/nomad.py

```diff
@@ -21,12 +23,34 @@ def name(check):
             )
             return check['Name']
 
-        spec = self.post(f'jobs/parse', {'JobHCL': hcl})
         for group in spec['TaskGroups'] or []:
             for task in group['Tasks'] or []:
                 for service in task['Services'] or []:
                     yield service['Name'], [name(check) for check in service['Checks'] or []]
 
+    def get_resources(self, spec):
+        """Generates (task, type, resources) tuples with resource stranzas for
+        the supplied job, with default values set for memory, disk and cpu."""
+
+        DEFAULTS = {
```

Comment:
Can we get this from nomad? They could differ depending on how it's built/configured.

Reply:
Even better: https://www.nomadproject.io/api/jobs.html#canonicalize and forget about the next comment